modules/ossm-security-cert-manage.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-security.adoc
+
+[id="ossm-cert-manage"]
+= Adding an external certificate authority key and certificate
+
+By default, {ProductName} generates self-signed root certificate and key, and uses them to sign the workload certificates. You can also use the operator-specified certificate and key to sign workload certificates, with operator-specified root certificate. This task demonstrates an example to plug certificates and key into {ProductShortName}.
+
+.Prerequisites
+
+* You must have installed {ProductName} with mutual TLS enabled to configure certificates.
+* You must use the certificates from link:https://github.com/maistra/istio/tree/maistra-2.0/samples/certs[Maistra repo]. 
+* You must deploy the Bookinfo sample application to verify the results with these instructions.
+
+[id="ossm-cert-manage-add-cert-key"]
+== Adding an existing certificate and key
+
+To use an existing signing (CA) certificate and key, you must create a chain of trust file that includes the CA certificate, key, and root certificate. You must use the following exact file names for each of the corresponding certificates. The CA certificate is called `ca-cert.pem`, the key is `ca-key.pem`, and the root certificate, which signs `ca-cert.pem`, is called `root-cert.pem`. If your workload uses intermediate certificates, you must specify them in a `cert-chain.pem` file. 
+
+Add the certificates to {ProductShortName} by following these steps. Save the certificates from the link:https://github.com/maistra/istio/tree/maistra-2.0/samples/certs[Maistra repo] locally and replace `<path>` with the path to your certificates.
+
+1. Create a secret `cacert` that includes the input files `ca-cert.pem`, `ca-key.pem`, `root-cert.pem` and `cert-chain.pem`.
++
+----
+$ oc create secret generic cacerts -n istio-system --from-file=<path>/ca-cert.pem \
+    --from-file=<path>/ca-key.pem --from-file=<path>/root-cert.pem \
+    --from-file=<path>/cert-chain.pem
+----
++
+2. In the `ServiceMeshControlPlane` resource set `global.mtls.enabled` to `true` and `security.selfSigned` set to `false`. {ProductShortName} reads the certificates and key from the secret-mount files.
```

Comment:
"resource" is a big vague here.  Could we say "In the `ServiceMeshControlPlane' configuration" or "In the `ServiceMeshControlPlane' YAML file" instead?

Reply:
Also, I feel like there is a missing step(s?) here.  If you're in the GUI you'd need to press the Reload button before the new certs are read.  Not sure what the equivalent oc command would be.

Reply:
@JStickler I left it that way on purpose. It was pointed out to me that you can update the SMCP several ways. I feel like if you're attempting this task, you've already mastered updating the SMCP and you'll do it in the way you've chosen.

However, I'm open to work with you to come up with a framework that we use here, and anywhere we need to update the SMCP. So there's a consistent way. I have some ideas.